src/Lib/Managers/SelectedBoundManager.vala

```diff
@@ -62,10 +64,15 @@ public class Akira.Lib.Managers.SelectedBoundManager : Object {
         if (selected_items.length () == 1) {
             var selected_item = selected_items.nth_data (0);
 
+            selected_item.store_relative_position ();
+
+            delta_x_accumulator = 0.0;
+            delta_y_accumulator = 0.0;
+
             initial_event_x = event_x;
             initial_event_y = event_y;
 
-            canvas.convert_to_item_space (selected_item, ref initial_event_x, ref initial_event_y);
+            // canvas.convert_to_item_space (selected_item, ref initial_event_x, ref initial_event_y);
```

Comment:
Why this is commented?

Reply:
Now dealing with item's coordinate system is not so easy nor obvious, since the item inside artboard are painted by the Artboard itself, so their coordinate system is bound to their parent. 

Here we need to convert the event x,y to the item rotation, since it is the only affine transform we do to the item, since `relative_x` and `relative_y` are the new coordinate system of the object relative to their parent. 

I think that, by introducing the Artboard feature, we need to rethink the way in which we convert events from the canvas to the object. 

I made some test, but eventually I opted to focus on more urgent tasks (like connecting all the "wires" together) and after take focus on the topic of dealing with affine transform and the canvas. 

I hope it does make sense.